numpy/add_newdocs.py

```diff
@@ -5741,9 +5747,13 @@ def luf(lamdaexpr, *args, **kwargs):
         The data-type used to represent the intermediate results. Defaults
         to the data-type of the output array if such is provided, or the
         the data-type of the input array if no output array is provided.
-    out : ndarray, optional
-        A location into which the result is stored. If not provided a
-        freshly-allocated array is returned.
+    out : ndarray, None, or tuple of ndarray and None, optional
+        A location into which the result is stored. If not provided or `None`,
+        a freshly-allocated array is returned. A tuple (possible only as a
+        keyword argument) must have length equal to the number of outputs.
```

Comment:
OK, thanks. Now I didn't follow this pattern, but did make a change, to add "if given as a keyword" (since we don't check for it if given as a positional element)